common/src/main/java/net/william278/huskhomes/random/NormalDistributionEngine.java

```diff
@@ -45,6 +46,12 @@ public NormalDistributionEngine(@NotNull HuskHomes plugin) {
         this.radius = plugin.getSettings().getRtp().getRegion();
         this.mean = plugin.getSettings().getRtp().getDistributionMean();
         this.standardDeviation = plugin.getSettings().getRtp().getDistributionStandardDeviation();
+
+        if (plugin.getSettings().getCrossServer().isEnabled()
+                && plugin.getSettings().getCrossServer().getBrokerType() != Broker.Type.REDIS) {
+            plugin.log(Level.WARNING,
+                    "CrossServer is enabled but is not using REDIS message broker, RTP will not work cross-server!");
+        }
```

Comment:
Yeah I think a toggle is more intuitive here (off by default, even, probably).

Probably also should adjust the message if that toggle is on - "Cross-server /rtp support has been disabled as a REDIS message broker is required for this feature."